rest/config.go

```diff
@@ -51,6 +54,10 @@ const (
 
 	// Default number of index replicas
 	DefaultNumIndexReplicas = uint(1)
+
+	// Take the LoggerCollateFlushTimeout and multiply it to give plenty of time to allow
+	// the log collation buffers to be flushed to outputs before exiting Sync Gateway.
+	logCollationBufferFlushDelay = 100 * base.LoggerCollateFlushTimeout
```

Comment:
Maybe move this constant to be defined beside LoggerCollateFlushTimeout to ensure any future modifications are done in step?

Switching to a fixed time (like 1s) might be more appropriate.